### PR TITLE
Use Peggy instead of deprecated Peg.js

### PR DESCRIPTION
**Pull request by sinistersnare**
Hi there, `Peg.js` is unmaintained, so I figure you all would appreciate if I replaced it with the drop-in replacement of Peggy. This is work I am breaking out of #918.

Peggy adds new features like source map support that we could use, although I do not include that in this change-set. It may be useful for debugging changes to the .pegjs file we have.e

### CODE REVIEW

File: packages/loot-core/webpack/webpack.browser.config.js [hunk 1]
```diff
@@ -28,11 +28,6 @@ module.exports = {
       path: 'path-browserify',
     },
   },
-  resolveLoader: {
-    alias: {
-      'pegjs-loader': require.resolve('pegjs-loader'),
```
Comment (MatissJanis) [inline packages/loot-core/webpack/webpack.browser.config.js:33]: Can we now do `yarn remove pegjs-loader`? :)

Comment (MatissJanis) [reply packages/loot-core/webpack/webpack.browser.config.js:33]: Ignore this.. I'm used to a single `package.json` in the root directory. So when I did not find it there I got confused.

But you already have done this. Thanks!

ps. - leaving this for Jed to review. Pegjs is used for goal templates and I'm not a user of this feature.

**Review by j-f1 — APPROVED** [review]
Nice improvement!

**Comment by j-f1** [comment]
I’ve found that editor support for Peggy syntax isn’t super great so I think it makes sense to hold off on using its language features for now.

**Pull request merged**